community/ibm-strongloop-dev/README.md

```diff
@@ -0,0 +1,90 @@
+# Strongloop
+
+[Strongloop](https://strongloop.com/)Open-source solutions for the API developer community
+
+```console
+$ helm install stable/ibm-strongloop-dev
+```
+
+## Prerequisites
+
+- Kubernetes 1.7+ 
+- Tiller 2.7.2 or later
+
+## Resources Required
+The chart deploys pods consuming minimum resources as specified in the resources configuration parameter (default: Memory: 200Mi, CPU: 100m)
+
+## PodSecurityPolicy Requirements
+This chart requires a PodSecurityPolicy to be bound to the target namespace prior to installation. Choose predefined ibm-anyuid-psp PodSecurityPolicy.
+
+## Introduction
+
+This chart bootstraps a [Strongloop](https://github.com/strongloop) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm intall --name my-release stable/ibm-strongloop-dev
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Chart Details
+This chart bootstraps a [Strongloop](https://hub.docker.com/r/ibmcom/strongloop-ppc64le/) deployment on a [Kubernetes](http://kubernetes.io) cluster
+
+
+## Configuration
+
+The following table lists the configurable parameters of the Strongloop chart and their default values.
```

Comment:
It is suggested here that the following three entries are the only configurable parameters, and yet, if I go to the GUI, I see many parameters that can be configured. If the intent is to only have these three be configurable, then the values.yaml needs to change. Otherwise, I'd expect that rows should be provided for each of the values I can configure in the GUI.

Reply:
Hi Robert,

Thanks for review.
We have done the requested changes. Please review and do let me know if you need any additional changes. 

Thanks,
Snehlata Mohite.

Reply:
If I expand the all parameters tab in the ICP UI, I see the following:
* Replica Count
* resources
* Tolerations
* Node Preference
* Repository
* Tag
* Docker image pull policy
* Service Type
* HTTP Port
* Enable Ingress
* annotations
* Path
* Virtual hosts
* TLS

I would expect that all of these be described in some fashion per [this guidance.](https://github.com/IBM/charts/blob/master/GUIDELINES.md#readmemd)